css-grid-1/grid-items/grid-item-containing-block-004.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Grid item sizing</title>
+<link rel="author" title="Tomek Wytrebowicz" href="mailto:tomalecpub@gmail.com">
+<link rel="help" href="http://www.w3.org/TR/css-grid-1/#grid-items" title="4. Grid Items">
+<meta name="assert" content="A grid item is sized within the containing block defined by its grid area">
```

Comment:
Or maybe change a little bit this text to be more descriptive.
Explaining that it's checking flexible tracks in a positioned grid container.